src/components/wallet-select/index.tsx

```diff
@@ -11,6 +12,24 @@ export interface Properties {
   onSelect?: (connector: WalletType) => void;
 }
 
+const providerConfig = {
```

Comment:
i think it maybe makes sense to just keep this as part of the existing wallet configuration? this is the kind of thing that the `wallets` collection in `./wallets.ts` is intended for.